### ISSUE TITLE
Pick a format for doc field

### ISSUE THREAD

**Issue by briandoconnor** [common-workflow-language/common-workflow-language#293]
It would be great if the doc field officially supported Markdown (or something else).  We encourage users of Dockstore to use Markdown but would prefer the CWL spec to specify which formats are supported (say text or Markdown).

**Comment by tetron** [comment]
I agree.  Currently it is it is de facto markdown but it should be specified.
